components/bl-image-list/src/index.js

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { ImageList, ImageListItem, ImageListItemBar } from './lib/mui';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const POS_BELOW = 'below';
+const TYPE_QUILTED = 'quilted';
+const IMAGE_HEIGHT_REDUCTION_FACTOR = 0.7;
+
+export default function ImageListComponent({ component, eventHandlers, elRef }) {
+  const { style, classList, display, itemData, listVariant, cols, rowHeight, gap } = component;
+
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    setItems(itemData);
+  }, [itemData]);
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div className={ cn('bl-customComponent-image-list', classList) } ref={ elRef } style={ style }>
+      <ImageList cols={ cols } gap={ gap } rowHeight={ rowHeight } variant={ listVariant }>
+
+        { items.map(item => (
+          <Item key={ item.img } item={ item } component={ component } eventHandlers={ eventHandlers }/>
```

Comment:
`item.img` is not uniq

Also, for the future(you should not change it now): `img` -> `imgURL`